com.unity.render-pipelines.core/Runtime/Common/ConstantBuffer.cs

```diff
@@ -105,14 +105,14 @@ public class ConstantBuffer
         /// <param name="shaderId">Shader porperty id to bind the constant buffer to.</param>
         public static void Set<CBType>(Material mat, int shaderId) where CBType : struct
         {
-            var cb = TypedConstantBuffer<CBType>.instance;
+            var cb = ConstantBufferSingleton<CBType>.instance;
 
             cb.Set(mat, shaderId);
         }
 
         /// <summary>
         /// Release all currently allocated constant buffers.
-        /// This needs to be called before shutting down the application.
+        /// This needs to be called before shutting down the application. 
```

Comment:
Maybe Update the doc to explain it's only for CBs used through the static API? (not sure of the right wording here)